package.json

```diff
@@ -7,17 +7,17 @@
     "build": "rollup -c",
     "dev": "rollup -c -w",
     "lint": "node node_modules/.bin/eslint --quiet .*.js src/",
-    "start": "static -c 0 -p 5000 dist"
+    "start": "./node_modules/.bin/serve dist"
```

Comment:
Please don't refer to node_modules. It breaks yarn2. Just 'serve dist' should do.